i18n/en.json

```diff
@@ -7521,5 +7521,9 @@
   {
     "id": "web.incoming_webhook.user.app_error",
     "translation": "Couldn't find the user"
+  },
+  {
+    "id": "cli.outgoing_webhook.inconsistent_state.app_error",
```

Comment:
This entry must be in the file in alphabetical order, so if you move it to the correct place (ordered alphabetically by id) is going to pass the i18n-check

Reply:
This is resolved.